### DISCUSSION TITLE
倍速快捷键是取消了吗

### DISCUSSION THREAD

**Discussion by FangchenHu** [the1812/Bilibili-Evolved#3578] [提问 / Q&A] [answered]
之前视频可用→按键来3倍速播放，现在按这个按钮变成进度条拖动，请问怎么变回原来的

**Comment by Geeyun-JY3** [comment, marked as answer]
长按倍速播放是 b 站本身的功能，你是不是开了 `快捷键扩展`？没禁用「前进」就会有你说的情况。

**Comment by Wkstr** [comment]
快捷键设置一下

**Comment by FangchenHu** [comment]
这是来自QQ邮箱的自动回复邮件。你好，我已收到你的邮件。我将在之后尽快给你回复。祝您身体健康，万事如意。
